examples/super_resolution/main.py

```diff
@@ -145,4 +149,6 @@ def checkpoint():
     print("Checkpoint saved to {}".format(model_out_path))
 
 
+ProgressBar().attach(trainer)
+
```

Comment:
```
if opt.debug:
    @trainer.on(Events.ITERATION_COMPLETED(every=log_interval))
    def terminate():
        trainer.terminate_epoch()
```
Hey @vfdev-5 I tried to terminate the current epoch using this(which gets stuck) or the `trainer.terminate()` for the debug mode in this example but the second method fails to create a checkpoint for the model. Can you give me a possible solution for this

Reply:
Simplest solution is to use `trainer.run(..., epoch_length=10)`, so the epoch will be equal to 10 iters. However, on iteration 11 (epoch 2) the datapoint will be the next one from the dataloader and wont be the same as at iteration 1.